WP3/D3.8/report.tex

```diff
@@ -143,15 +145,17 @@ \section{Changes in scope of the deliverable}\label{changes-to-deliverable}}
 To this end, the \Sage project is already providing implicit cross-platform
 build testing for many projects; see the next section for details.
 
-\TODO{Say something here about conda-forge?}
+\TODO{Say something here about conda-forge? jrueth would be happy to say
+something about conda-forge. What did embray have in mind?}
```

Comment:
I can do it.  I just wanted to mention what you said, that some have found it a viable way to achieve cross-platform testing using their build tools.

But clarify something for me, if you get a chance:  I seem to recall you saying that they had some kind of special backroom deal with Travis CI to ensure that amount of resources and/or build priority they need?  Or did mix that up with something else?  If true, do you have a source on that?  I tried searching but didn't come up with much specific (and some of the search results are confused because Continuum was founded by Travis Oliphant :)

Reply:
Also, do you know of any specific projects that are using conda builds for multi-platform testing?  If I can't cite any specific examples I might not bother mentioning it at all.

Reply:
I don't have a source and I don't even know whether it's true. It's just that their travis (or Circle I forget) builds are sometimes saying something like "you exceeded your quota by 11000%, please upgrade to paid plan".

I think cysignals was the first project attempting something like that.